libs/langchain/langchain/retrievers/kendra.py

```diff
@@ -336,6 +345,8 @@ class AmazonKendraRetriever(BaseRetriever):
     page_content_formatter: Callable[[ResultItem], str] = combined_text
     client: Any
     user_context: Optional[Dict] = None
+    score_confidence: Optional[str] = "LOW"
+    score_confidence_dict = {"LOW": 1, "MEDIUM": 2, "HIGH": 3, "VERY_HIGH": 4}
```

Comment:
I see there is also `NOT_AVAILABLE` (https://docs.aws.amazon.com/kendra/latest/APIReference/API_ScoreAttributes.html). Any reason it is not handled here?

Reply:
There is not much information provided about status `NOT_AVAILABLE `. As per my observation there are 4 confidence returned Kendra never seen `NOT_AVAILABLE `.  May be `NOT_AVAILABLE ` for results which don't have any score confidence available in that case we are going to skip that records if filter is set by user.

Reply:
Make sense. Changed score confidence to numeric value.